mathics/builtin/numbers/exptrig.py

```diff
@@ -21,11 +25,140 @@
 from mathics.core.symbols import Symbol
 
 from mathics.builtin.arithmetic import _MPMathFunction
-from mathics.builtin.atomic.numbers import Fold
 
 from mathics.core.attributes import listable, numeric_function, protected
 
 
+class Fold(object):
```

Comment:
In Python3 the `(object)` part isn't needed, and usually is hidden.

Reply:
BTW, `Fold` is also a Builtin. Maybe this is an opportunity to rename it to `_Fold` or something else...